qubes-kde-dom0/qubes-kde-dom0.spec

```diff
@@ -2,13 +2,16 @@
 
 %if 0%{?qubes_builder}
 %define _sourcedir %(pwd)/qubes-kde-dom0
+%define qubes_kde_dom0_version %(cat version)
+%define breeze_version %(cat version_breeze)
+%define breeze_epoch %(cat epoch_breeze)
```

Comment:
For now, I placed the version for breeze inside the spec file rather than causing `version*` file proliferation.

KDE 5 has a number of different versioning schemes, and without finalizing the KDE 5 customizations, I did not feel comfortable removing the file named `version`.
* plasma-breeze 5.5.4
* plasma-desktop 5.4.2
* kf5-plasma 5.21.0
* kde-runtime 15.12.3